bottom-bar/src/main/java/com/roughike/bottombar/OverrideTabSelectionListener.java

```diff
@@ -18,12 +18,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-public interface TabOverrideListener {
+public interface OverrideTabSelectionListener {
```

Comment:
This whole PR is really good stuff! But I'm thinking if this could be named better.

Alternative suggestions:

- TabSelectionOverrider
- TabSelectionInterceptor
- ???